src/puppetlabs/puppetdb/mq.clj

```diff
@@ -205,7 +205,8 @@
 (extend String
   ToJmsMessage
   {:-to-jms-message (fn [x properties session]
-                      (let [msg (.createTextMessage session x)]
+                      (let [msg (.createBytesMessage session)]
+                        (.writeBytes msg (.getBytes x "UTF-8"))
```

Comment:
This call and the createBytesMessage above it use reflection, probably something we should get cleaned up.